# HG changeset patch
# User Victor Ruehle <ruehle@votca.org>
# Date 1359258783 25200
# Branch stable
# Node ID f26d7ee778d666babe7f58ae3770b4f3d997aa99
# Parent  d10dec78492a78ad6709e686416615fc8b43c896
fix build with boost-1.53 (fixes issue 131)

diff -r d10dec78492a -r f26d7ee778d6 include/votca/tools/tokenizer.h
--- a/include/votca/tools/tokenizer.h	Sun Dec 02 21:09:27 2012 -0700
+++ b/include/votca/tools/tokenizer.h	Sat Jan 26 20:53:03 2013 -0700
@@ -93,8 +93,10 @@
         std::vector<std::string> tmp;
         ToVector(tmp);
         v.resize(tmp.size());
-        transform(tmp.begin(), tmp.end(), v.begin(),
-            boost::lexical_cast<T, std::string>);
+        typename std::vector<T>::iterator viter = v.begin();
+        typename std::vector<std::string>::iterator iter;
+        for(iter = tmp.begin(); iter!=tmp.end(); ++iter, ++viter)
+            *viter = boost::lexical_cast<T, std::string>(*iter);
     }
     
 private:
